src/pathfinder/yapf/yapf_common.hpp

```diff
@@ -60,12 +67,12 @@ class CYapfOriginTileTwoWayT
 	typedef typename Node::Key Key;               ///< key to hash tables
 
 protected:
-	TileIndex   m_orgTile;                        ///< first origin tile
-	Trackdir    m_orgTd;                          ///< first origin trackdir
-	TileIndex   m_revTile;                        ///< second (reversed) origin tile
-	Trackdir    m_revTd;                          ///< second (reversed) origin trackdir
-	int         m_reverse_penalty;                ///< penalty to be added for using the reversed origin
-	bool        m_treat_first_red_two_way_signal_as_eol; ///< in some cases (leaving station) we need to handle first two-way signal differently
+	TileIndex origin_tile; ///< first origin tile
+	Trackdir origin_td; ///< first origin trackdir
+	TileIndex reversed_tile; ///< second (reversed) origin tile
+	Trackdir reversed_td; ///< second (reversed) origin trackdir
```

Comment:
```suggestion
	TileIndex reverse_tile; ///< second (reverse) origin tile
	Trackdir reverse_td; ///< second (reverse) origin trackdir
```

A bit more correct IMO, and move in line with reverse_penalty.